lumber-install.js

```diff
@@ -17,6 +18,10 @@ program
   // NOTICE: Load the environment variables from the .env to avoid always asking for the database
   //         connection information.
   dotenv.load();
+
+  eventSender.command = 'install';
+  [eventSender.appName] = program.args;
```

Comment:
[Misc] pretty weird syntax. I don't really get its use.

Reply:
It's the same as `eventSender.appName = program.args[0];`. ESLint force you to use this syntax

Reply:
Good to know